go.graphics.swing/src/main/java/go/graphics/swing/sound/SwingSoundPlayer.java

```diff
@@ -28,9 +23,14 @@
 import javax.sound.sampled.LineUnavailableException;
 import javax.sound.sampled.SourceDataLine;
 
+import go.graphics.sound.ForgettingQueue;
+import go.graphics.sound.ForgettingQueue.Sound;
+import go.graphics.sound.ISoundDataRetriever;
+import go.graphics.sound.SoundPlayer;
+
 public class SwingSoundPlayer implements SoundPlayer {
 	private static final int BUFFER_SIZE = 4048 * 4;
-	private static final int SOUND_THREADS = 3;
+	private static final int SOUND_THREADS = 30;
```

Comment:
3 is definitely not enough, 10 might be a compromise and 30 sounds good - it is up to you what you select here - just make a sound check on a bit more crowded map